### ISSUE TITLE
Implemetation details incoherence

### ISSUE THREAD

**Issue by khatanas** [facebookresearch/CutLER#30]
Hello,

I am a bit confused about which learning rate you used for which training stage. In section 3.5., in the **Detector** paragraph it is written:
> "We train the detector on ImageNet with initial masks and bounding boxes for 160K iterations with a batch size of 16." 

What is the learning rate here ? 

A little further it is written:
> "We then optimize the detector for 160K iterations using SGD with a learning rate of 0.005, which is decreased by 5 after 80K iterations, and a batch size of 16"

Assuming that these 2 sentences refer to the training stages where you are using DropLoss:
1. When I check the *cascade_mask_rcnn_R_50_FPN.yaml* file, the BASE_LR parameter is set to 0.01 (whereas USE_DROPLOSS: True, and MAX_ITER: 160K).
1. Did you trained only once before moving to the self-training stages (do these 2 sentences refer to the same training stage) ?

Then, in the **Self-training** paragraph:
> "We optimize the detector using SGD with a learning rate of 0.01 for 80K iterations."

* When I check the *cascade_mask_rcnn_R_50_FPN_self_train.yaml* file, the BASE_LR parameter is set to 0.005 (whereas USE_DROPLOSS: False, and MAX_ITER: 80K).

Please let me know.
